benchmarks/torchbench/voirfile.py

```diff
@@ -1,11 +1,28 @@
 def instrument_probe(ov):
-    # ov.probe(
-    #     "/torchbenchmark.util.framework.vision.model_factory/TorchVisionModel/train(data as batch) > #endloop_data as step"
-    # ).give()
-
+    ov.probe("/run/run_one_step > gpu_time").give("gpu_time")
+    ov.probe("/run/run_one_step > cpu_dispatch_time").give("cpu_dispatch_time")
+    ov.probe("/run/run_one_step > cpu_walltime").give("cpu_walltime")
+    ov.probe("/run/run_one_step > tflops").give("tflops")
```

Comment:
Use `give()` instead of `give("gpu_time")` etc. because the stream returned by probe is already made out of dictionaries with that key. The `.give(name)` syntax is if you have a stream of non-dictionaries and you want to assign a key to them.